src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder result = new StringBuilder();
+        while (value > 0) {
+            int a = value % 2;
```

Comment:
try not to create variable "a" but pass value directly into insert() method, that might be redundant

Reply:
done